Firebase.podspec

```diff
@@ -48,8 +49,7 @@ Simplify your app development, grow your user base, and monetize more effectivel
         'HEADER_SEARCH_PATHS' => "$(inherited) ${PODS_ROOT}/Firebase/CoreOnly/Sources"
       }
     end
-    # Standard platforms PLUS watchOS.
-    ss.ios.deployment_target = '10.0'
+    ss.ios.deployment_target = '9.0'
```

Comment:
Is this 9.0? Should we move the version dependencies of other Firebase SDKs to be 9.0 (Currently we moved Fireperf to 10.0)

Reply:
Yes - Analytics and AdMob are still 9. Everything else should now be at 10.

Reply:
Perfect. Good then!